website/cue/reference/remap/functions.cue

```diff
@@ -24,6 +24,7 @@ remap: {
 		}
 		internal_failure_reasons: [...string]
 		examples?: [remap.#Example, ...remap.#Example]
+		deprecated: bool | *false
```

Comment:
Note for the interviewer: Does this make sense? How we usually mark functions as deprecated?

Reply:
This seems reasonable. I see you added the "deprecation message" in the description, which seems fine to me.

If we want, we could use this flag to visually indicate deprecated functions on https://vector.dev/docs/reference/vrl/functions/. This would be done around here:

https://github.com/vectordotdev/vector/blob/600f8191a8fe169eb38c429958dd59714349acb4/website/layouts/partials/data.html#L663-L835

I'd suggest adding another badge next to the `infallible` badge that says `deprecated` and is yellow.

Reply:
Nice, I will add a "deprecated" badge in this PR.

Reply:
You can see how it looks:

- https://deploy-preview-18150--vector-project.netlify.app/docs/reference/vrl/functions/#to_timestamp
- https://deploy-preview-18150--vector-project.netlify.app/docs/reference/vrl/expressions/#function-call-characteristics